jaeger-core/src/main/java/com/uber/jaeger/senders/HttpSender.java

```diff
@@ -44,7 +46,7 @@
   private static final TProtocolFactory PROTOCOL_FACTORY = new Factory();
   private static final TSerializer SERIALIZER = new TSerializer(PROTOCOL_FACTORY);
 
-  private final HttpClient httpClient = new DefaultHttpClient();
+  private final HttpClient httpClient = new DefaultHttpClient(new PoolingClientConnectionManager());
```

Comment:
Ideally, it would be better to give users a choice. We could split it into a separate module.

> I think brave is using it and it's smaller than apache.

https://github.com/openzipkin/zipkin-reporter-java , there is okhttp, urlconnection...